Compentio.SourceMapper.Generator/Processors/ClassProcessorStrategy.cs

```diff
@@ -64,7 +64,7 @@ private string GeneratePartialClassMethods(IMapperMetadata sourceMetadata)
 
         private string GeneratePartialClassMethod(IMethodMetadata methodMetadata)
         {
-            return $"public abstract {AttributesMatchers.GetInverseMethodFullName(methodMetadata)};";
+            return $"public abstract {GetInverseMethodFullName(methodMetadata)};";
```

Comment:
It seems to be like methodMetedata property

Reply:
Updated.